client/web/compose/src/components/PageBlocks/Navigation/Base.vue

```diff
@@ -0,0 +1,125 @@
+          :disabled="list.options.disabled"
+          :variant="list.options.backgroundColor"
+          :style="{ order: index, color: list.options.textColor }"
+          :link-attrs="{ style: `color: ${list.options.textColor}` }"
+          :target="selectTargetOption(list.options.item.target)"
+          :text="list.options.item.label"
+          :right="list.options.align === 'right'"
+          :left="list.options.align === 'left'"
+          class="nav-item-dropdown"
+        >
+          <template v-if="!list.options.item.displaySubPages && list.type !== 'dropdown'">
+            <span>
+              {{ list.options.item.label }}
+            </span>
+          </template>
+
+          <template v-if="(list.type === 'dropdown')">
+            <b-dropdown-item
+              v-for="(dropdown, dIndex) in list.options.item.dropdown"
+              :key="`dropdown-${dIndex}`"
+              :to="dropdown.url"
+              :disabled="list.options.disabled"
+              :style="{ order: dIndex * 2 }"
+            >
+              {{ dropdown.label }}
+            </b-dropdown-item>
+
+            <b-dropdown-divider
+              v-for="(dropdown, dIndex) in list.options.dropdown"
+              v-show="dropdown.delimiter"
+              :key="`dropdown-divider-${dIndex}`"
+              :style="{ order: dIndex + 1 }"
+            />
+          </template>
+
+          <template v-if="(list.type === 'compose' && list.options.item.displaySubPages)">
+            <b-dropdown-item
+              v-for="(dropdown, dIndex) in getSubPages(list.options.item.pageID)"
+              :key="`dropdown-${dIndex}`"
+              :disabled="list.options.disabled"
+              :to="dropdown.pageID"
+            >
+              {{ dropdown.title }}
+            </b-dropdown-item>
+          </template>
+        </component>
+      </b-nav>
+    </div>
+  </wrap>
+</template>
+<script>
+import { mapGetters } from 'vuex'
+import base from '../base'
+
+export default {
+  extends: base,
+
+  computed: {
+    ...mapGetters({
+      pages: 'page/set',
+    }),
+  },
+
+  methods: {
+    displayNavItemDrop (list) {
+      if (list.options.item.label && (!list.options.item.displaySubPages && list.type !== 'dropdown')) {
+        return 'b-nav-item'
+      }
+
+      if (list.type === 'dropdown') {
+        return 'b-nav-item-dropdown'
+      }
+
+      if (list.type === 'compose' && list.options.item.displaySubPages) {
+        return 'b-nav-item-dropdown'
+      }
+    },
+    getSubPages (selfID) {
+      return this.pages.filter(value => value.selfID === selfID) || []
+    },
+    selectTargetOption (target) {
+      switch (target) {
+        case 'sameTab':
+          return '_self'
+        case 'newTab':
+          return '_blank'
+        case 'newWindow':
+          return '_blank'
+      }
+    },
+  },
+}
+</script>
+
+<style>
+.nav-item-dropdown .dropdown-menu.show {
+  display: grid;
+}
+
+.navigation-page-block-item .nav-item {
```

Comment:
Same as the above class, you can have it on the component like you do now. So you just need one class